msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

```diff
@@ -1,63 +1,85 @@
+- **Enhanced security.** The client application does not need to manage the refresh token which can be used to obtain new authentication tokens without user consent.
 - **Feature support.** With the help of the broker developers can access rich OS and service capabilities such as Windows Hello, conditional access policies, and FIDO keys without writing extra scaffolding code.
 - **System integration.** Applications that use the broker plug-and-play with the built-in account picker, allowing the user to quickly pick an existing account instead of reentering the same credentials over and over.
 
-With the help of a broker you need to write less code to handle token-related logic while also being able to use more advanced functionality, such as [Proof-of-Possession tokens](../../advanced/proof-of-possession-tokens.md). Moving forward, our team is continuing to invest in making sure that brokers are the de facto approach to authenticate with MSAL-based applications, where possible.
+With the help of a broker you need to write less code to handle token-related logic while also being able to use more advanced functionality, such as [Proof-of-Possession tokens](../../advanced/proof-of-possession-tokens.md). Moving forward, the MSAL team is continuing to invest in making sure that brokers are the de facto approach to authenticate inside applications.
 
 ## Improved Windows broker experience
 
-Recently, an updated authentication broker experience was made available for Windows systems. The new broker is written in C++, is well tested, and is significantly more performant and secure. One of the biggest consumers of the new broker experience is the Microsoft 365 suite of apps.
+Latest Windows releases include an updated WAM. The new broker is written in C++, is well-tested, and is significantly more performant and secure. One of the biggest consumers of the new broker experience is the Microsoft 365 suite of apps.
 
 > [!IMPORTANT]
-> With the introduction of the updated broker, we have also updated MSAL.NET to expose its capabilities under a simplified API. Starting with version 4.52.0, the improved broker experience in MSAL.NET will replace the existing broker flows by default.
+> With the introduction of the updated broker, we have also updated MSAL.NET to expose its capabilities under a simplified API. Starting with version 4.52.0, the new broker will be the default in MSAL.NET.
 
-The improved WAM broker fixes a number of issues with the legacy WAM implementation and provides other benefits:
+The new WAM fixes a number of issues with the legacy implementation and provides other benefits, including:
 
 - New implementation is more stable, easier to add new features, and has less chance of regressions.
-- Works in apps that are run-as-admin.
-- Adds support for Proof-of-Possession tokens.
-- Fixes assembly size issues.
+- Works in apps that are ran under the Administrator user context.
```

Comment:
```suggestion
- Works in apps that are run under the Administrator user context.
```

Reply:
I am actually curious about the right grammar here, @gladjohn 😄 Maybe the right word here is "executed."